src/estimagic/differentiation/derivatives.py

```diff
@@ -31,29 +38,24 @@ def first_derivative(
     error_handling="continue",
     batch_evaluator="joblib",
     return_func_value=False,
-    return_info=True,
+    return_info=False,
     key=None,
 ):
     """Evaluate first derivative of func at params according to method and step options.
 
     Internally, the function is converted such that it maps from a 1d array to a 1d
-    array. Then the Jacobian of that function is calculated. The resulting derivative
-    estimate is always a :class:`numpy.ndarray`.
+    array. Then the Jacobian of that function is calculated.
 
-    The parameters and the function output can be pandas objects (Series or DataFrames
-    with value column). In that case the output of first_derivative is also a pandas
-    object and with appropriate index and columns.
+    The parameters and the function output can be general pytrees. By default the
```

Comment:
We should call it `estimagic-pytrees` instead of general pytrees and link to corresponding section of the EEP. General pytrees makes it sound like a pytree of strings would work.